crates/adapters/src/transport/kafka/ft/input.rs

```diff
@@ -429,7 +448,7 @@ impl KafkaFtInputReaderInner {
                 if !kafka_paused {
                     self.pause_partitions()
                         .map_err(|error| self.refine_error(error).1)?;
-                    for receiver in receivers.iter() {
+                    for (_, receiver) in receivers.iter() {
```

Comment:
.values()?

Reply:
I see there's a few other places where this might be better, I'll stop mentioning them.